src/components/LinkButtons/LinkButtons.module.scss

```diff
@@ -0,0 +1,34 @@
+@import '../_variables.scss';
+
+.wrap {
+  margin-top: 40px;
+  display: flex;
```

Comment:
Try to use the outside-in CSS organization. We try to follow these guidelines in all our projects, so it's easier for everyone to read CSS. Please resolve this in all scss

https://webdesign.tutsplus.com/articles/outside-in-ordering-css-properties-by-importance--cms-21685

Reply:
okay, will fix